sigoptlite/sigoptlitetest/endpoints/test_observation.py

```diff
@@ -310,11 +322,13 @@ def test_assignments_observations_log_transform_bounds(self, connection):
     ]
     e = connection.experiments().create(**experiment_meta)
 
-    with pytest.raises(SigOptException):
+    with pytest.raises(SigOptException) as exception_info:
       connection.experiments(e.id).observations().create(
         assignments=dict(x1=-1),
         values=[{"name": "y1", "value": 1.23}],
       )
+    msg = "Assignment must be positive for log-transformed parameter x1"
```

Comment:
could use f string here and a few other places

Reply:
the parameter is hard coded in the test, so there's no need to use a variable